roles/certs/idm-host-cert/tasks/create-directories.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+- file:
+    path: "{{ target_cert_files.host_cert | dirname }}"
+    recurse: yes
+- file:
+    path: "{{ target_cert_files.fullchain | dirname }}"
```

Comment:
@oybed Don't think we need this here since IdM doesn't handle fullchain as you mentioned.

Reply:
Good point. Will correct.